state-reconstruct-fetcher/src/constants.rs

```diff
@@ -20,7 +20,7 @@ pub mod storage {
     pub const DEFAULT_DB_NAME: &str = "db";
 
     /// The name of the state file.
-    pub const STATE_FILE_NAME: &str = "StateSnapshot.json";
+    pub const INNER_DB_NAME: &str = "inner_db";
```

Comment:
IMHO that goes to the question of why we have 2 databases, which I don't get... :-) Is there some setup in which we have multiple inner databases for the same outer one? Or an index to key mapping w/o the other snapshot data, or vice versa?

Reply:
> why we have 2 databases, which I don't get

Essentially, we need to use two databases as the state tree crate we're using is an abstraction over another RocksDB database. There _might_ be ways to store it all in one database, but I'm not aware of such a solution. Feel free to research this one later though, would solve a lot of sync issues.

> Is there some setup in which we have multiple inner databases for the same outer one?

I don't think this would ever be necessary (right now), the only use case I can see is if we were to separate index-to-key mappings and the fetcher progress metadata. Since the index-to-key mappings is only read/written from the consumers side, while fetcher metadata is only read/written from the producer. But again, if there's no need to do this now, we shouldn't.